node/src/chain_spec/battery_park.rs

```diff
@@ -41,7 +41,9 @@ pub fn battery_park_config(
                 wasm_binary,
             )
         },
-        vec![],
+        vec![
+            "/ip4/139.162.171.58/tcp/30333/p2p/12D3KooWPvu5rpH2FNYnAmiQ8X8XqkMiuSFTjH2jwMCSjoam7RGQ".parse().unwrap()
```

Comment:
When passing a bad peer id, `unwrap` will panic the current program execution but since this path is about node initialization, it shouldn't be a problem. If desirable, it is also possible to `.parse().map_err(|_| "Invalid boot node id")?`.